src/averageAges.js

```diff
@@ -37,7 +41,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const mothersList = people.map(person => person.mother);
```

Comment:
you can move this variable inside `if` statement to not do unnecessary calculation when `withChildren` is `false`